test/startservers.py

```diff
@@ -63,17 +47,17 @@ def start(race_detection):
     global processes
     forward()
     progs = [
-        'boulder-wfe --config %s' % get_config('wfe'),
-        'boulder-ra --config %s' % get_config('ra'),
-        'boulder-sa --config %s' % get_config('sa'),
-        'boulder-ca --config %s' % get_config('ca'),
-        'boulder-va --config %s' % get_config('va'),
-        'boulder-publisher --config %s' % get_config('publisher'),
-        'ocsp-updater --config %s' % get_config('ocsp-updater'),
-        'ocsp-responder --config %s' % get_config('ocsp-responder'),
-        'ct-test-srv --config %s' % get_config('ct-test-srv'),
-        'dns-test-srv --config %s' % get_config('dns-test-srv'),
-        'mail-test-srv --config %s' % get_config('mail-test-srv'),
+        'boulder-wfe --config %s' % os.path.join(default_config_dir, "wfe.json"),
+        'boulder-ra --config %s' % os.path.join(default_config_dir, "ra.json"),
+        'boulder-sa --config %s' % os.path.join(default_config_dir, "sa.json"),
+        'boulder-ca --config %s' % os.path.join(default_config_dir, "ca.json"),
+        'boulder-va --config %s' % os.path.join(default_config_dir, "va.json"),
+        'boulder-publisher --config %s' % os.path.join(default_config_dir, "publisher.json"),
+        'ocsp-updater --config %s' % os.path.join(default_config_dir, "ocsp-updater.json"),
+        'ocsp-responder --config %s' % os.path.join(default_config_dir, "ocsp-responder.json"),
+        'ct-test-srv',
+        'dns-test-srv',
+        'mail-test-srv',
```

Comment:
_Note for other reviewers_: `ct-test-srv`, `dns-test-srv` and `mail-test-srv` do not have a `--config` option or call `flag.Parse()` and so the previous `--config` argument supplied for these three `progs` in `start` were superfluous and misleading.